test/Williamson2_ShallowWaterExplicit.jl

```diff
@@ -0,0 +1,114 @@
+module Williamson2_ShallowWaterExplicit
```

Comment:
Add this file to `runtests.jl` ... otherwise we do not test this in Github actions on each push